conans/test/cmake_test.py

```diff
@@ -1,10 +1,18 @@
+import shutil
+import sys
+import tempfile
 import unittest
 from conans.model.settings import Settings
 from conans.client.conf import default_settings_yml
 from conans.client.cmake import CMake
 
 
 class CMakeTest(unittest.TestCase):
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
Oh, damn it, you are right (once again :) 
It is fine as-is then, no big problem, nothing to change then, thanks!